docs/documentation/plugins/index.html

```diff
@@ -1482,6 +1482,44 @@ <h3 id="skeleton">Skeleton</h3>
         }
     }
 
+    // If the plugin is a button
+    function buildButtonIcon() {
+        if (!document.querySelector("#trumbowyg-myplugin")) {
```

Comment:
I prefer the early return way, aka if the element exists, return; then, the rest of the function runs only if we need to create the button. It prevent from having the whole function in a if

Reply:
Now that you mention it, so do I. See: fa7ead9